pyspectral/utils.py

```diff
@@ -550,5 +551,11 @@ def are_instruments_identical(name1, name2):
         return True
     if name1 == INSTRUMENT_TRANSLATION_DASH2SLASH.get(name2):
         return True
+    if type(name1) is list and type(name2) is str:
```

Comment:
I think it will work, but the function name is not really representing what this does anymore. Could/should this be refactored a bit to handle both cases (lists versus name as str)?
At least the doc string needs to explain

Reply:
Hopefully I have resolved this now!